packages/kit/src/runtime/server/page/render.js

```diff
@@ -273,12 +273,13 @@ export async function render_response({
 		);
 
 		for (const path of included_modulepreloads) {
-			// we use modulepreload with the Link header for Chrome, along with
-			// <link rel="preload"> for Safari. This results in the fastest loading in
-			// the most used browsers, with no double-loading. Note that we need to use
-			// .mjs extensions for `preload` to behave like `modulepreload` in Chrome
+			// see the kit.output.preloadStrategy option for details on why we have multiple options here
 			link_header_preloads.add(`<${encodeURI(path)}>; rel="modulepreload"; nopush`);
-			head += `\n\t\t<link rel="preload" as="script" crossorigin="anonymous" href="${path}">`;
+			if (options.preload_strategy !== 'modulepreload') {
```

Comment:
hmm, is the rel=modulepreload link always added ? shouldn't it be either one or the other?

Reply:
The modulepreload equivalent is the header links above. Since it's a header it doesn't hurt I think, but you're right we could put it inside an if block

Reply:
Chrome doesn't respect `preload` headers, only the `<link>`. This might be a bug in Chrome, but I haven't got round to investigating and/or filing a ticket. Either way, I think we need to keep the `modulepreload` link headers for Chrome's benefit

Reply:
Wait, really? This means that chrome didn't get preload behavior previously, too, because that logic I implemented is basically the one we had prior to your PR which introduced the link preload. Mhmmm.
Isn't there also a related issue about the link headers being too large and crashing something? Do we need another "linkheaders" option?

Reply:
Not sure I follow? we previously added `modulepreload` links to the HTTP header and still do (and should)